client/app/visualizations/table/renderer.less

```diff
@@ -0,0 +1,53 @@
+.table-visualization-container {
+  .ant-pagination.ant-table-pagination {
+    float: none;
+    display: block;
+    text-align: center;
```

Comment:
Perhaps also add `margin-bottom: 0` so there's no needless spacing in the widget and full screen view.